h1_syscalls/Cargo.toml

```diff
@@ -0,0 +1,35 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+[package]
+name = "h1_syscalls"
+version = "0.1.0"
+authors = ["Philip Levis <plevis@google.com>", "Oskar Senft <osk@google.com"]
+edition = "2018"
+
+[profile.dev]
+panic = "abort"
+lto = false
+opt-level = "z"
+debug = true
+
+[profile.release]
+panic = "abort"
+lto = true
+opt-level = "z"
+debug = true
```

Comment:
The Cargo profiles (`profile.dev`, `profile.release`) only take effect when this crate is built directly. Because this is a Rust library crate, it isn't useful on its own, so I don't think these profiles will ever be used.

Reply:
Thanks, I didn't realize that. I incorrectly copied that from golf2 instead of taking it from h1. Fixed.